### PR TITLE
Solr4

### PR DESCRIPTION
**Pull request by ebbesson**
Adding solr 4 support and creates a specific solr 3.X stage

### CODE REVIEW

File: stages/out/solr-out-3.6/CHANGES.txt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+refs #1344
+Removed slf4j-dependency
+
+Integration of SolrOutputStage feature into trunk fixes #1344.
+
+refs #1445
+changing package names. Huge change. Fourth project, one more to go
+
+refs #1445
+changing artifact, group and dependency names
```
Comment (jwestberg) [inline stages/out/solr-out-3.6/CHANGES.txt:49]: What refs are these?

Comment (ebbesson) [inline stages/out/solr-out-3.6/CHANGES.txt:49]: I believe those are leftovers prior-to oos-ing Hydra

**Comment by rgraneru** [comment]
I've confirmed that the 3.6 version still works in this build.

**Comment by laserval** [comment]
Due to API changes and slacking committer (myself :( ), I've fixed up this pull request in #169.

Closing this and will get the new pull request merged.

**Pull request closed without merging**